### PR TITLE
Activity Selection in C

### PR DESCRIPTION
**Pull request by yashaswibiyahut**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

**YES**

### Description

Issue no: #826
Activity selection in c.
**_Approach_** : Greedy Algorithm.
**_Time Complexity_** -
The complexity of this problem is O(n log n) when the list is not sorted.
When the sorted list is provided the complexity will be O(n).

_Below is the output of given code._
### Checklist

- [x]  I've read the contribution guidelines.
- [x]  I've checked the issue list before deciding what to submit.
- [x]  I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by yashaswibiyahut** [comment]
C__Users_yashaswi Biyahut_Desktop_Activity exe 9_3_2020 2_11_31 PM (2)
Output of the above code.

**Comment by yashaswibiyahut** [comment]
kindly review this PR @Kajol-Kumari @SKAUL05 @siddharth25pandey @iamrajiv @HarshCasper

File: C/cp/Activity_selection.c [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+{
+	int temp;
+	temp=*x;
+	*x=*y;
+	*y=temp;
+}
+//Main function start
+int main()
+{
+	int num,start[20],finish[20],i,j,prev=0,count=1;
```
Comment (siddharth25pandey) [inline C/cp/Activity_selection.c]: Use Dynamic memory allocation for arrays

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@yashaswibiyahut A small change is required, rest is fine

File: C/cp/Activity_selection.c (hunk 1 above)
Comment (yashaswibiyahut) [reply C/cp/Activity_selection.c]: Done @siddharth25pandey kindly review it.

File: C/cp/Activity_selection.c [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+			if(finish[j]>finish[j+1])
+			{
+				//call swap function
+				//pass by address
+				swap(&finish[j],&finish[j+1]);
+				swap(&start[j],&start[j+1]);
+			}
+		}
+	}
+	//condition for Activity selection
```
Comment (Kajol-Kumari) [inline C/cp/Activity_selection.c:48]: please use multi line comments here.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Please provide one line space before each function definition and incorporate the suggested changes, Rest LGTM :+1:

File: C/cp/Activity_selection.c (hunk 2 above)
Comment (yashaswibiyahut) [reply C/cp/Activity_selection.c:48]: by mistake i deleted the branch before merging thats y i am not be able to make changes. Can you tell me how to do or i have to make a new PR? @Kajol-Kumari

**Pull request closed without merging**